terraform/digitalocean/main.tf

```diff
@@ -60,26 +60,35 @@ module "etcd_cert" {
   ca_private_key_pem = "${module.ca.ca_private_key_pem}"
 }
 
-module "apiserver_certs" {
-  source             = "../certs/kubernetes/apiserver"
-  ca_cert_pem        = "${module.ca.ca_cert_pem}"
-  ca_private_key_pem = "${module.ca.ca_private_key_pem}"
-  ip_addresses       = "${join(",", digitalocean_droplet.master.*.ipv4_address)}"
-  master_count       = "${var.masters}"
-}
-
-module "worker_certs" {
-  source             = "../certs/kubernetes/worker"
-  ca_cert_pem        = "${module.ca.ca_cert_pem}"
-  ca_private_key_pem = "${module.ca.ca_private_key_pem}"
-  ip_addresses       = "${join(",", digitalocean_droplet.worker.*.ipv4_address)}"
-  worker_count       = "${var.workers}"
-}
-
-module "admin_cert" {
-  source             = "../certs/kubernetes/admin"
-  ca_cert_pem        = "${module.ca.ca_cert_pem}"
-  ca_private_key_pem = "${module.ca.ca_private_key_pem}"
+module "kube_apiserver_certs" {
+  source                = "github.com/Capgemini/tf_tls/kubernetes/apiserver"
+  ca_cert_pem           = "${module.ca.ca_cert_pem}"
+  ca_private_key_pem    = "${module.ca.ca_private_key_pem}"
+  ip_addresses          = "${join(",", digitalocean_droplet.master.*.ipv4_address)}"
+  master_count          = "${var.masters}"
+  validity_period_hours = "8760"
+  early_renewal_hours   = "720"
+  ssh_user              = "core"
+  ssh_private_key       = "${tls_private_key.ssh.private_key_pem}"
+}
+
+module "kube_worker_certs" {
+  source                = "github.com/Capgemini/tf_tls/kubernetes/worker"
+  ca_cert_pem           = "${module.ca.ca_cert_pem}"
+  ca_private_key_pem    = "${module.ca.ca_private_key_pem}"
+  ip_addresses          = "${join(",", digitalocean_droplet.worker.*.ipv4_address)}"
```

Comment:
You are assuming  you have a list in var.ip_addresses here https://github.com/Capgemini/tf_tls/blob/master/kubernetes/worker/deploy.tf#L16
But a string is being passed